src/pages/CreateInvoice.tsx

```diff
@@ -38,19 +46,30 @@ import { deserializeAmount, serializeAmount } from '~/core/serializers/serialize
 import {
   AddOnForInvoiceEditTaxDialogFragmentDoc,
   CurrencyEnum,
+  FetchDraftInvoiceTaxesMutation,
+  LagoApiError,
   TaxInfosForCreateInvoiceFragment,
   useCreateInvoiceMutation,
+  useFetchDraftInvoiceTaxesMutation,
   useGetAddonListForInfoiceLazyQuery,
   useGetInfosForCreateInvoiceQuery,
 } from '~/generated/graphql'
 import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { useIntegrations } from '~/hooks/useIntegrations'
 import { useSalesForceConfig } from '~/hooks/useSalesForceConfig'
 import ErrorImage from '~/public/images/maneki/error.svg'
 import { Card, HEADER_TABLE_HEIGHT, MenuPopper, PageHeader, theme } from '~/styles'
 import { StickySubmitBar } from '~/styles/mainObjectsForm'
 
 const CELL_HEIGHT = 68
 
+enum LocalTaxProviderErrorsEnum {
+  CurrencyCodeNotSupported = 'text_17238318811308wqpult4i7r',
+  CustomerAddressError = 'text_1723831881130x4cfh6qr6o8',
+  ProductExternalIdUnknown = 'text_1723831881130g8hv6qzqe57',
+  GenericErrorMessage = 'text_17238318811307ghoc4v7mt9',
+}
```

Comment:
Maybe we can make it generic because I'll need the same mapping in the invoice details alert message wdyt?

Reply:
Done!